docs/tx_lifecycle.md

```diff
@@ -0,0 +1,119 @@
+
+##### 1b. ... or from L1 (via the Delayed Inbox).
+
+Alternatively, a client can send a message to the Sequencer by signing and publishing an L1 transaction in the Arbitrum chain's Delayed Inbox. This functionality is most commonly used for depositing ETH or tokens via a bridge.
+
+**See**:
+
+- [Retryables](arbos/ArbOS#retryables)
+- "The Sequencer"
+- "Token Bridge"
+
+### 2. Sequencer orders transaction (off-chain)
+
+Upon receiving a transaction, the Sequencer will:
+
+- Order it in its off-chain Inbox
+- Locally execute it using the Arbitrum Nitro VM (including collecting/allocating L1 and L2 fees, etc.)
+- "Instantly" give a transaction receipt to the client ("instant" in that it doesn't require any additional on-chain confirmations, and typically shouldn't take more than a second or two).
+
+**See**:
+
+- [ArbOS](arbos/ArbOS)
+- [Geth](arbos/ArbOS)
+- [L1 pricing](arbos/L1_Pricing) / [L2 Gas](Gas)
+
+#### ~ ~ ~ FINALITY CHECK: Trusted / Soft Confirmation ~ ~ ~
+
+At this phase, the client's acceptance of finality relies on some degree of trust in the Sequencer. I.e., a malicious/faulty Sequencer could deviate between what it promised in the transaction receipt and what is ultimately published in a batch (see phase 3).
+
+Note that even a malicious/faulty Sequencer can only, at worst, reorder or temporarily delay transactions; it cannot, e.g., forge a client's transaction or propose an invalid state update. Given the degree of trust in the Sequencer at phase 2, we sometimes refer to the "instant" receipt that the Sequencer provides as a "soft confirmation."
+
+#### 3. Sequencer posts transaction in a batch (on-chain)
+
+The Sequencer will eventually post a batch of L2 transactions which includes our client's transaction onto the underlying L1 (as calldata); under normal conditions, the Sequencer will post batches [every few minutes](https://arbiscan.io/batches).
+
+##### 3a. What if the Sequencer never includes our transaction?
+
+Even if the Sequencer never includes our transaction in a batch, the client can include it in the L2 by posting in the delayed inbox and the "force including" it after some delay period (currently ~24 hours on Arbitrum One).
+
+**See:**
+
+- "The Sequencer / Censorship Resistance."
+
+#### ~ ~ ~ FINALITY CHECK: Ethereum-Equivalent Finality! ~ ~ ~
+
+At this stage, assuming that a client believes there to be at least one well behaved active Arbitrum validator (recall that in Arbitrum Rollup, validating is permissionless), the client can treat their transaction's finality as equivalent to an ordinary Ethereum transaction. In other words, their transaction has L1-level security. This means the client should use whatever finality heuristic they use for Ethereum transactions (i.e., waiting on L1 block confirmations, etc.) This also means that a client uncomfortable with the trust model of the Sequencer's soft confirmations (phase 2) can simply wait for the Sequencer to post their transaction in a batch (phase 3).
```

Comment:
Suggest clarifying here that your tx is as final *as the L1 tx that records the batch it is in*.